lib/bootstrap/environment.rb

```diff
@@ -56,6 +56,31 @@ def pattern_path(path)
   end
 end
 
+def flatten_hash(h,f="",g={})
```

Comment:
:+1: I was struggling to decide where to put them, I'll experiment moving them to `LogStash::Util`